x-pack/plugins/fleet/server/services/api_keys/index.ts

```diff
@@ -22,17 +22,8 @@ export async function generateOutputApiKey(
       cluster: ['monitor'],
       index: [
         {
-          names: [
-            'logs-*',
-            'metrics-*',
-            'traces-*',
-            '.ds-logs-*',
-            '.ds-metrics-*',
-            '.ds-traces-*',
-            '.logs-endpoint.diagnostic.collection-*',
-            '.ds-.logs-endpoint.diagnostic.collection-*',
-          ],
-          privileges: ['write', 'create_index', 'indices:admin/auto_create'],
+          names: ['logs-*', 'metrics-*', 'traces-*', '.logs-endpoint.diagnostic.collection-*'],
+          privileges: ['auto_configure', 'create_doc', 'indices:admin/auto_create'],
```

Comment:
Do we need both 'indices:admin/auto_create' and 'auto_configure'?  I believe the latter implies the former.

The core team has discouraged us from using the non-documented privileges.